tests-e2e/cypress/integration/zoom_setup_wizard.js

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+// <reference path="../support/index.d.ts" />
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+const defaultZoomConfig = {
+    "accountlevelapp": false,
+    "apikey": "",
+    "apisecret": "",
+    "enableoauth": true,
+    "encryptionkey": "",
+    "oauthclientid": "",
+    "oauthclientsecret": "",
+    "webhooksecret": "",
+    "zoomapiurl": "",
+    "zoomurl": ""
+}
+
+describe('Zoom setup wizard', () => {
+    beforeEach(() => {
+        // # Login as sysadmin
+        cy.apiAdminLogin();
+
+        cy.apiDeleteAllPostsInDirectChannel('sysadmin', 'zoom');
+
+        cy.apiUpdateConfig({
+            PluginSettings: {
+                Plugins: {
+                    zoom: defaultZoomConfig,
+                },
+            },
+        });
+
+        cy.apiDisablePluginById('zoom');
+        cy.apiEnablePluginById('zoom');
+
+        cy.visit('/test/messages/@zoom');
```

Comment:
Just for my knowledge: How do we know the test team exist and the zoom plugin is installed?

Reply:
Whoops I added the team creation to the GitHub plugin wizard e2e test but not the Zoom wizard https://github.com/mattermost/mattermost-plugin-github/blob/c22ad7188c35beddca3d3766e96e118b34bbeead/tests-e2e/cypress/integration/setup_wizard_spec.js#L39

@larkox The zoom plugin will be installed by CI, similar to how we're doing with the Apps plugin CI